packages/core/http/core-http-server/src/router/router.ts

```diff
@@ -88,6 +88,11 @@ export interface IRouter<Context extends RequestHandlerContextBase = RequestHand
   getRoutes: () => RouterRoute[];
 }
 
+export type RouterRouteHandler = (
```

Comment:
nit: This type is used in a public interface, we need to export it at the `index.ts` level